### PR TITLE
#91263 [User] [Room] Quit a room

### PR DESCRIPTION
**Pull request by kopitop**

### CODE REVIEW

File: app/Repositories/Eloquent/RoomRepository.php [hunk 1]
```diff
@@ -225,4 +225,69 @@ public function beginPlay($id)
...
+        //If user is not in the room 
+        if (!$data['info']->isJoining()) {
+            throw new Exception;
+        }
+
+        //Update player slots
+        if ($data['info']->isDrawer()) {
+            $data['info']->fill([
+                    'drawer_id' => null,
+                ]);
```
Comment (dung13890) [inline app/Repositories/Eloquent/RoomRepository.php]: convention

File: app/Repositories/Eloquent/RoomRepository.php [hunk 2]
```diff
@@ -225,4 +225,69 @@ public function beginPlay($id)
...
+        } else {
+            $data['info']->fill([
+                    'guesser_id' => null,
```
Comment (dung13890) [inline app/Repositories/Eloquent/RoomRepository.php]: convention

**Comment by DungNQ-Framgia** [comment]
ok

**Comment by dung13890** [comment]
ok

**Pull request merged**